src/SDKs/Compute/Compute.Tests/VMScaleSetTests/VMScaleSetServiceFabricScenarioTests.cs

```diff
@@ -33,6 +33,26 @@ private void TestVMScaleSetServiceFabricImpl(MockContext context)
             string vmssName = "crptesthtn39hve";
 
             var response = m_CrpClient.VirtualMachineScaleSets.ForceRecoveryServiceFabricPlatformUpdateDomainWalk(rgName, vmssName, 0);
+
+#if NET46
```

Comment:
Your test is never built against .NET 46, so this is essentially dead code.

Reply:
@shahabhijeet  Yes.  This is a dead code, but in our repo, we build tests using NET46 because moving to netcore is not done in our repo.  Since we copy test codes from our internal repo to github, we need to change it every time.  That is why I added this part.